br/README.md

```diff
@@ -1,131 +1,148 @@
 
-# How many rows do we get again? Expected to be 100000 rows.
-mysql -uroot -htidb -P4000 -E -e "SELECT COUNT(*) FROM test.usertable"
+### Best practice
+The following are some recommended operations for using `TiKV-BR` for backup and restoration:
+- It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
+- `TiKV-BR` supports restore on clusters of different topologies. However, the online applications will be greatly impacted during the restore operation. It is recommended that you perform restore during the off-peak hours or use rate-limit to limit the rate.
+- It is recommended that you execute multiple backup operations serially. Running different backup operations in parallel reduces backup performance and also affects the online application.
+- It is recommended that you execute multiple restore operations serially. Running different restore operations in parallel increases Region conflicts and also reduces restore performance.
+- `TiKV-BR` supports checksum between `TiKV` cluster and backup files after backup or restore with the config `--checksum=true`. Note that, if checksum is enabled, please make sure no data is changed or `TTL` expired in `TiKV` cluster during backup or restore.
+- TiKV-BR supports [`api-version`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#api-version-%E4%BB%8E-v610-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) conversion from V1 to V2 with config `--dst-api-version V2`. Then restore the backup files to APIV2 `TiKV` cluster
 
-# Test S3 compatible storage (MinIO).
-# Create a bucket to save backup by mc (a MinIO Client).
-mc config host add minio $S3_ENDPOINT $MINIO_ACCESS_KEY $MINIO_SECRET_KEY && \
-mc mb minio/mybucket
+### TiKV-BR Command Line Description
+A br command consists of sub-commands, options, and parameters.
 
-# Backup to S3 compatible storage.
-bin/br backup full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+- Sub-command: the characters without - or --, including `backup`, `restore`, `raw` and `help`.
+- Option: the characters that start with - or --.
+- Parameter: the characters that immediately follow behind and are passed to the sub-command or the option.
+#### Backup Raw Data
+To back up the cluster raw data, use the `tikv-br backup raw` command. To get help on this command, execute `tikv-br backup raw -h` or `tikv-br backup raw --help`.
+For example, backup raw data in TiKV cluster to `/tmp/backup` directory.
 
-# Drop database and restore!
-mysql -uroot -htidb -P4000 -E -e "DROP DATABASE test; SHOW DATABASES;" && \
-bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
-    --s3.endpoint="$S3_ENDPOINT"
+```
+tikv-br backup raw \
+    --pd "&{PDIP}:2379" \
+    -s "s3://backup-data/2022-09-16/" \
+    --s3.endpoint "s3://backup-data/2022-09-16/" \
```

Comment:
It seems that it's not necessary to specify `--s3.endpoint` when `-s` using the `s3://` schema.